core/unit_test/CMakeLists.txt

```diff
@@ -182,6 +182,7 @@ foreach(Tag Threads;Serial;OpenMP;Cuda;HPX;OpenMPTarget;OpenACC;HIP;SYCL)
 
     set(${Tag}_SOURCES1B)
     set(${Tag}_TESTNAMES1B
+        AtomicAccessor
```

Comment:
Same comment about the name.  I see that we did not follow that convention for the space-aware accessor but I find it unfortunate.  I find that this helps navigating unit tests and understanding better the structure of the code base.